server/test/services/applications/PdfExporterServiceTest.java

```diff
@@ -53,10 +53,7 @@ public void generateApplicationPdf() throws IOException {
       assertThat(AnnotationAction.get(PdfName.S)).isEqualTo(PdfName.URI);
       PdfString link = AnnotationAction.getAsString(PdfName.URI);
       assertThat(link.toString())
-          .isEqualTo(
-              "http://localhost:9000/admin/programs/"
-                  + applicationOne.getProgram().id
-                  + "/files/my-file-key");
+          .isEqualTo("http://localhost:9000/admin/applicant-files/my-file-key");
```

Comment:
So for acled file paths - no program id in the url. Am I going correctly?

Reply:
Yep. (You can see FileController for the methods that end up being called. The old adminShow takes a program ID, but doesn't use it and just calls into the new one. So all this change does is call directly into the new one, and thus doesn't need the program ID).